src/CsvParser.php

```diff
@@ -8,6 +8,10 @@
  */
 class CsvParser implements \Iterator
 {
+    /**
+     * @var string
+     */
+    static private $bom = "\xef\xbb\xbf";
```

Comment:
imo, this is an internal technical detail and should therefore not be public - constants cannot be private/protected in php :disappointed: